main/rpc/index.js

```diff
@@ -137,6 +139,14 @@ const rpc = {
   updateRequest(reqId, actionId, data, cb = () => {}) {
     accounts.updateRequest(reqId, actionId, data)
   },
+  updateTypedSignatureRequest(id, data) {
```

Comment:
does the existing `updateRequest` method work for this?

Reply:
which part of it falls over? would be nice to unify these code paths if at all possible